app/views/providers/check_benefits/index.html.erb

```diff
@@ -1,31 +1,39 @@
-<%= content_for(:panel) do %>
-  <div class="govuk-panel govuk-panel--confirmation">
-    <h2 class="govuk-panel__title">
+<div class="interruption-panel">
+  <%= page_template page_title: '', column_width: 'full'  do %>
+<%
+# blank title - title dealt with below
+%>
+    <h1 class="govuk-heading-l">
       <%= @legal_aid_application.applicant.full_name %>
-      <br />
       <%= t "#{result_namespace}.title" %>
-    </h2>
-  </div>
-<% end %>
-
-<%= page_template page_title: t('.heading_1') do %>
-
-  <p class="govuk-body">
-    <%= t "#{result_namespace}.body" %>
-  </p>
-
-  <h3 class="govuk-heading-m">
-    <%= t '.heading_2' %>
-  </h3>
-
-  <p class="govuk-body">
-    <%= t "#{result_namespace}.next" %>
-  </p>
-
-  <%= next_action_buttons_with_form(
-        url: providers_legal_aid_application_check_benefit_path,
-        method: :patch,
-        show_draft: true,
-        continue_button_text: t('generic.continue')
-      ) %>
-<% end %>
+    </h1>
+    <div class="govuk-grid-column-two-thirds">
+      <% if result_namespace == '.negative_result' %>
+        <p class="govuk-body">
+          <%= t "#{result_namespace}.paragraph" %>
+        </p>
```

Comment:
So the paragraph appears above the `<h2>` on a negative result, and below it on a positive result. That seems a little weird and adds quite a lot of complication to the page.

Reply:
This was my way of reducing the number of variables in `providers.yml`.  If that isn't important, I can make them separate things.

Reply:
Nope - now I'm totally confused as to what you are doing.

Let me be a little clearer. The code above - lines 12-14 (that only appears if `result_namespace == '.negative_result'`):
```erb
<p class="govuk-body">
  <%= t "#{result_namespace}.paragraph" %>
</p>
```
is exactly the same as the code between lines 20-22 in a section that only appears if `result_namespace == '.positive_result'`

As the `.next_heading` code is below the first case, and above the second, and assuming the only `result_namespace` values are `'.positive_result'` and `.negative_result` all this effectively does is move the heading below or above the paragraph code respectively.

If as I expect, the heading can be above the paragraph in both conditions this whole section (lines 11-30) can be simplified to:
```erb
<h2 class="govuk-heading-m">
  <%= t '.next_heading' %>
</h2>
<p class="govuk-body">
  <%= t "#{result_namespace}.paragraph" %>
</p>
<% if result_namespace == '.positive_result' %>
  <ul class="govuk-list govuk-list--bullet">
    <li><%= t "#{result_namespace}.next_1" %></li>
    <li><%= t "#{result_namespace}.next_2" %></li>
  </ul>
<% else %>
  <p class="govuk-body"><%= t "#{result_namespace}.next_1" %></p>
  <p class="govuk-body"><%= t "#{result_namespace}.next_2" %></p>
<% end %>
```

Reply:
No, the _next heading_ is above the paragraph in one situation and below in the other situation.  

The paragraphs have different text, I have called them both _paragraph_ to reduce the number of variables in the yml file. 

I'll re-work this all tomorrow, splitting them into two variables, and hopefully it'll be clearer then.  Will let you know when it's done for you to look at again.